test.c

```diff
@@ -0,0 +1,19 @@
+#include "CurrentRange.h"
+#include <assert.h>
+
+int main()
+{
+int *chkrange;
+
+int array[] = {3, 3, 5, 4, 10, 11, 12};
```

Comment:
Test cases need to be strengthened to check with various test set. Single value , multiple values, repetitive values etc.
Please add test cases.